tests/pr_test.go

```diff
@@ -100,3 +100,28 @@ func TestRunCompleteUpgradeExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunSecureSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  "solutions/secure",
+		Region:        "us-south",
+		Prefix:        "els-sr-da",
+		ResourceGroup: resourceGroup,
+	})
+
+	options.TerraformVars = map[string]interface{}{
+		"access_tags":                permanentResources["accessTags"],
+		"existing_kms_instance_guid": permanentResources["hpcs_south"],
+		"kms_key_crn":                permanentResources["hpcs_south_root_key_crn"],
+		"existing_resource_group":    true,
+		"resource_group_name":        options.ResourceGroup,
```

Comment:
remove `existing_resource_group` (so it defaults to false) and set `"resource_group_name": options.Preifx,` so we test the DA creating a new resource group